packages/adapter/src/lib/adapter/adapter.js

```diff
@@ -1531,7 +1535,7 @@ function Adapter(options) {
             } else {
                 logger && logger.warn(this.namespaceLog + ' slow connection to objects DB. Still waiting ...');
             }
-        }, (config.objects.connectTimeout || 2000) * 3); // Because we do not connect only anymore, give it a bit more time
+        }, config.objects.connectTimeout * 3); // Because we do not connect only anymore, give it a bit more time
```

Comment:
With the above change, this can be simplified because `connectTimeout` is definitely defined.

Reply:
In fact *2 would be also be sufficient now ...